### PR TITLE
The software craftsman book page

### PR DESCRIPTION
**Pull request by ghost**
the software craftsman page

### CODE REVIEW

File: _i18n/es.yml [hunk 1]
```diff
@@ -63,7 +63,7 @@ index-software_craftsmanship:
 index-software_craftsmanship-book:
    title: The Software Craftsman&#58; de nuestro Co-fundador Sandro Mancuso
    subtitle: El libro es descrito como "una enciclopedia sobre los comportamientos, atributos, y la estructura de una organización que se adhiere a los principios de Software Craftsmanship". Una lectura obligatoria para cualquiera que desee comprender mejor los valores detrás del movimiento.
-   link_title: Descubre más en Good Reads ...
+   link_title: Check the book contents ...
```
Comment (jukecraft) [inline _i18n/es.yml]: that's not spanish :)

File: publications/the-software-craftsman.html [hunk 2]
```diff
@@ -0,0 +1,156 @@
...
+				<p>This book is an encyclopedia on the behavior, attributes, and structure of an organization striving to grow in professionalism and adhere to the principles of Software Craftsmanship.</p>
+				<span>Robert C. Martin</span>
+			</div>
+			</div>
+		</div>
+	</div>	
+
+  	<div class="row">
+  		<div class="container md-col-6">
+	  		<p>The book covers a wide range of subjects related to our profession and is full of advices and personal stories which illustrate the current state of our industry, how things could be better, and what developers can do to bring more professionalism, pragmatism, and pride to our industry.</p>
```
Comment (jukecraft) [inline publications/the-software-craftsman.html]: full of advice

File: publications/the-software-craftsman.html [hunk 3]
```diff
@@ -0,0 +1,156 @@
...
+	  		<p>The book clearly resonates with Robert C. Martin who has provided a beautifully written foreword. He summarises the book as an <span>encyclopedia on the behaviour, attributes, and structure of an organization striving to grow in professionalism and adhere to the principles of Software Craftsmanship.</span></p>
```
Comment (jukecraft) [inline publications/the-software-craftsman.html]: that's a duplication to the quote above

File: publications/the-software-craftsman.html [hunk 4]
```diff
@@ -0,0 +1,156 @@
...
+	  		<p>The preface has a very inspirational story about how Sandro encountered his first mentor and how that relationship has shaped his personal and professional life.</p>
+  		</div>
+	</div>	
+
+	<div class="row">
+		<div class="title-header">
+      		<h2>Part I - Ideology and Attitude</h2>
+		<p>In the first part, Sandro defines the Software Craftsmanship ideology and the attitude expected from true software craftsmen.</p>
```
Comment (jukecraft) [inline publications/the-software-craftsman.html:47]: can we find another way to say "ideology and attitude"?

File: publications/the-software-craftsman.html [hunk 5]
```diff
@@ -0,0 +1,156 @@
...
+    	</div>
+  	</div>
+
+  	<div class="row">
+		<div class="container col-md-6 col-sm-6">
+			<ul class="list-inline">
+				<li>
+					<span><b>Chapter 1 </b> Software Development in the Twenty-First Century</span>
+					<p>Describes how wrongly seniority is measured in most companies and explains how developers should evolve and behave in order to cope with more modern ways of working.</p>
```
Comment (jukecraft) [inline publications/the-software-craftsman.html:56]: Describes how seniority is measured the wrong way in most companies...
... in order to work in a modern environment

File: publications/the-software-craftsman.html [hunk 6]
```diff
@@ -0,0 +1,156 @@
...
+				</li>
+				<li>	
+					<span><b>Chapter 2 </b> Agile<span>
+					<p>Describes the problems with most Agile adoptions and how Software Craftsmanship can help to fix it by providing a good balance between process and technical practices.
```
Comment (jukecraft) [inline publications/the-software-craftsman.html]: ... can help to solve them

File: publications/the-software-craftsman.html [hunk 7]
```diff
@@ -0,0 +1,156 @@
...
+  		<div class="container col-md-6 col-sm-6">
+			<ul class="list-inline">
+				<li><span><b>Chapter 5 </b> Heroes, Goodwill, and Professionalism</span> 
+					<p>Addresses  how to deal with pressure and tight deadlines in a professional manner.</p></li>
+				<li><span><b>Chapter 6 </b> Working Software</span>
+					<p>Describes the problems of low quality software and how blaming the business for it is not an option. With a different attitude, developers could make things much better.</p></li>
+				<li><span><b>Chapter 7 </b> Technical Practices</span> 
+					<p>Helps developers to understand and communicate the business value associated to technical practices, increasing the chances to have them adopted. It also talks about pragmatism and accountability.</p></li>
+				<li><span><b>Chapter 8 </b> The Long Road</span> 
+					<p>This chapter is about the determination that is needed to have a successful career. This is one of Sandro’s favourite chapter and we won’t spoil it telling you more. :) </p></li>
```
Comment (jukecraft) [inline publications/the-software-craftsman.html]: ... we won't spoil it by telling you more

File: publications/the-software-craftsman.html [hunk 8]
```diff
@@ -0,0 +1,156 @@
...
+			</ul>
+		</div>
+  	</div>
+
+	<div class="row">
+		<div class="title-header">
+      		<h2>Part II - A Full Transformation</h2>
+			<p>In the second half, Sandro focus on bringing Software Craftsmanship principles and values to organisations.</p>
```
Comment (jukecraft) [inline publications/the-software-craftsman.html]: Sandro focusses

File: publications/the-software-craftsman.html [hunk 9]
```diff
@@ -0,0 +1,156 @@
...
+  	<div class="row">
+		<div class="container col-md-6 col-sm-6">
+			<ul class="list-inline" >
+				<li>
+						<span><b>Chapter 9 </b> Recruitment</span> 
+						<p>For most companies recruitment is broken. This chapters explains how to attract great developers, write job descriptions, and have *proactive* recruitment.</p>
+				</li>
+				<li>
+						<span><b>Chapter 10 </b> Interviewing Software Craftsmen</span> 
+						<p>States that interview process is a business negotiation and provides advice for both companies and candidates to reach mutually beneficial agreements.</p>
```
Comment (jukecraft) [inline publications/the-software-craftsman.html]: States that the interview process

File: publications/the-software-craftsman.html [hunk 10]
```diff
@@ -0,0 +1,156 @@
...
+		</div>
+  		<div class="container col-md-6 col-sm-6">
+			<ul class="list-inline" >
+				<li>
+						<span><b>Chapter 13 </b> Culture of Learning</span> 
+						<p>Provides many ideas and examples for creating and nurturing a culture of learning. It also shows that any developer can do it even with no support from management.</p>
+				</li>				
+				<li>
+						<span><b>Chapter 14 </b> Driving Technical Changes</span> 
+						<p>Identifies different patterns of skepticism and offers many ideas on how to overcome them. It also provides advices on how to establish trust and how to increase the chances of convincing people with opposite views.</p>
```
Comment (jukecraft) [inline publications/the-software-craftsman.html]: is also provides suggestions / it also provides advice

File: publications/the-software-craftsman.html [hunk 11]
```diff
@@ -0,0 +1,156 @@
...
+				</li>
+				<li>
+						<span><b>Chapter 15 </b> Pragmatic Craftsmanship</span> 
+						<p>Craftsmanship without pragmatism is not craftsmanship. Quality is not expensive—lack of skills is. This is an important chapter that busts a few myths about Craftsmanship being expensive and slow.</p>
```
Comment (jukecraft) [inline publications/the-software-craftsman.html:125]: Quality is not expensive, but lack of skills is

**Review by jukecraft — COMMENTED** [review]
some suggestions for changes in wording

**Comment by ghost** [comment]
@sandromancuso can you have a look on the suggestions made by Franzi?

Cheers

File: publications/the-software-craftsman.html (hunk 4 above)
Comment (ghost) [reply publications/the-software-craftsman.html:47]: not really

File: publications/the-software-craftsman.html (hunk 5 above)
Comment (ghost) [reply publications/the-software-craftsman.html:56]: not changed

**Pull request merged**